modules/configuring-a-network-policy-for-loki.adoc

```diff
@@ -0,0 +1,56 @@
+:_newdoc-version: 2.18.4
+:_template-generated: 2025-10-17
+:_mod-docs-content-type: PROCEDURE
+
+[id="configuring-a-network-policy-for-loki_{context}"]
+= Configuring a network policy for Loki
+
+Enable or disable the deployment of `NetowrkPolicies` per LokiStack by setting the `networkPolicies` field. 
+
+.Prerequisites
+* You have administrator permissions.
+* You have installed the {oc-first}.
+* You have installed the {loki-op}.
+* You have created a `LokiStack` custom resource (CR).
+
+.Procedure
+. Update the `LokiStack` CR:
++
+[source,yaml]
+----
+apiVersion: loki.grafana.com/v1
+kind: LokiStack
+metadata:
+  name: logging-loki
+  namespace: openshift-logging
+spec:
+  size: 1x.small
+  storage:
+    schemas:
+    - version: v13
+      effectiveDate: "<yyyy>-<mm>-<dd>"
+    secret:
+      name: logging-loki-s3
+      type: s3
+  storageClassName: <storage_class_name>
+  tenants:
+    mode: openshift-logging
+  networkPolicies:
+    ruleSet: RestrictIngressEgress
+----
++
+You can set one of the following values for the `spec.networkPolicies.ruleSet` field:
++
+`None`::
+{loki-op} will not create any network policy.
```

Comment:
suggest to make it to 'will not deploy' instead of create.

Reply:
done